test/blackbox-tests/test-cases/dune-cache/trim/run.t

```diff
@@ -125,25 +125,25 @@ they are part of the same rule.
 
   $ mkdir -p $PWD/.xdg-cache/dune/db/temp
   $ mkdir -p $PWD/.xdg-cache/dune/db/runtime
-  $ mkdir -p $PWD/.xdg-cache/dune/db/v1
   $ mkdir -p $PWD/.xdg-cache/dune/db/v2
   $ mkdir -p $PWD/.xdg-cache/dune/db/v3
-  $ mkdir -p $PWD/.xdg-cache/dune/db/files/v1
+  $ mkdir -p $PWD/.xdg-cache/dune/db/v4
   $ mkdir -p $PWD/.xdg-cache/dune/db/files/v2
   $ mkdir -p $PWD/.xdg-cache/dune/db/files/v3
-  $ mkdir -p $PWD/.xdg-cache/dune/db/meta/v1
+  $ mkdir -p $PWD/.xdg-cache/dune/db/files/v4
```

Comment:
v4 ? Was it not bumped to just v3 ?

Reply:
I was just trying to be too clever, testing the the case of future versions. I'll simplify this test a bit.